map/framework.hpp

```diff
@@ -669,7 +669,7 @@ class Framework : public PositionProvider,
 
 public:
   void AllowTransliteration(bool allowTranslit);
-  bool LoadTransliteration();
+  static bool LoadTransliteration();
```

Comment:
Why make only this one static?

Reply:
Transliteration (same as 3D for example) should be the state of Framework (inner drape engine).
The current "settings" implementation (not the best one) allows to make this (and some others) functions static, but I'd not do it. Tomorrow, they will be state modifiers.

Reply:
Yep I know. And a bunch of other member functions could be static as well. But I don't think either that they should be. And also on the caller side, `LoadTransliteration` is called on an object.

Reply:
Agree, removed static. @vng @Ferenc- PTAL